src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/service/LimitedPartnershipServiceTest.java

```diff
@@ -35,8 +35,8 @@ public class LimitedPartnershipServiceTest {
     private static final String USER_ID = "xbJf0l";
     private static final String SUBMISSION_ID = "abc-123";
     private static final String REQUEST_ID = "fd4gld5h3jhh";
-    private static final String TRANSACTION_ID = "12321123";
-
+    private static final String TRANSACTION_ID = "txn-456";
+   
```

Comment:
Spaces here and an additional line added on 124. We went through checking for these kinds of things @HatiDev . I know they are minor, but if you are missing these kinds of things in GitHub after pushing, perhaps you are missing other things?

Reply:
spaces removed

Reply:
There are now spaces on line 50 of the `PartnershipController` class. Did you check in GitHub after pushing?

Reply:
Partnership class reverted